packages/boot/test/fixtures/application.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {RestApplication} from '@loopback/rest';
+import {ApplicationConfig} from '@loopback/core';
+
+/* tslint:disable:no-unused-variable */
+// Binding and Booter imports are required to infer types for BootMixin!
+import {BootMixin, Booter, Binding} from '../../index';
+/* tslint:enable:no-unused-variable */
+
+export class ControllerBooterApp extends BootMixin(RestApplication) {
+  constructor(options?: ApplicationConfig) {
+    super(options);
+    this.projectRoot = __dirname;
+  }
+
+  async start() {
+    await super.start();
```

Comment:
This method can be removed, since it's effectively a no-op now. Thoughts?

Reply:
Good catch!